src/i18n/l10n/en.ftl

```diff
@@ -615,14 +615,15 @@ Username-or-Email = Username or Email
 
 Visually-search-iNaturalist-data = Visually search iNaturalist’s wealth of data. Search by a taxon in a location
 
-Welcome-back = Welcome back,
-
 Whenever-you-get-internet-connection-you-can-upload = Whenever you get internet connection, you can upload your observations to iNaturalist.
 
 Which-traditional-projects-can-add-your-observations = Which traditional projects can add your observations?
 
 Who-can-add-observation-fields-to-my-observations = Who can add observation fields to my observations?
 
+# Welcome user back to app
+Welcome-user = <0>Welcome back,</0><1>{$userHandle}</1>
```

Comment:
Not sure if this is correct syntax to indicate different elements in fluent.

Reply:
Well it works, so that's good. Looks like you can also add some semantic specificity, e.g. 

fluent:
```fluent
Welcome-user = <plain>Welcome back,</plain><bold>{$userHandle}</bold>
```

javascript:
```javascript
<Trans
  i18nKey="Welcome-user"
  parent={View}
  values={{ userHandle: User.userHandle( currentUser ) }}
  components={{
    plain: <Body1 className="mt-1 text-3xl" />,
    bold: <Body1 className="mb-1 text-3xl font-semibold" />
  }}
/>
```

But I'm not sure that's necessary here with only two interpolations. It might help translators understand what those interpolations will do, but it could just as easily mislead them into translating the variable names (I've seen the latter happen a lot).